Silicon/CommonSocPkg/Include/Library/BootGuardLib20.h

```diff
@@ -72,4 +72,32 @@ ExtendBtGuardEvents(
   IN BOOT_GUARD_INFO *BtGuardInfo
   );
 
```

Comment:
This lib should be in APL silicon folder only. Its not common used lib.

Reply:
Same comment as above.

Reply:
See comment above for resolution.